### ISSUE TITLE
[Accessibility checks] image-alt missing on Bug BBQ page

### ISSUE THREAD

**Issue by rivaquiroga** [r-devel/r-devel.github.io#26]
`image-alt` missing on bug-bbq welcome page: https://contributor.r-project.org/events/bug-bbq/welcome.html: 

- "bug_list.png" 
- "bug-bbq_GitHub.png"

**Comment by PatriLoto** [comment]
I'll add the alt-text to the images
